.github/workflows/reusable-build-docs.yaml

```diff
@@ -68,8 +74,19 @@ jobs:
       run: echo VERSION="${PACKAGE_VERSION}" >> "${GITHUB_ENV}"
 
     - name: Build the Ansible community package docs
+      if: env.REDIRECTS == 'true'
+      run: >-
+        make webdocs EXTRA_TAGS="-t redirects" ANSIBLE_VERSION="${{
+          env.PACKAGE_VERSION != 'devel' && env.PACKAGE_VERSION || ''
+        }}"
+      working-directory: build-directory/docs/docsite
+
+    - name: Build the Ansible community package docs
+      if: env.REDIRECTS == 'false'
```

Comment:
```suggestion
```

Reply:
@webknjaz I believe this should be conditional based on the inputs. For some builds, such as deploying to gh-pages, you probably don't want to generate redirects.

Also if we remove the condition here, shouldn't we remove the `generate-redirects` input?